net-wireless/horst/horst-5.1.ebuild

```diff
@@ -18,15 +19,16 @@ RDEPEND="
 	sys-libs/ncurses:0
 	pcap? ( net-libs/libpcap )
 "
-TDEPEND="
-	test? ( sys-devel/sparse )
-"
 DEPEND="
 	${RDEPEND}
-	${TDEPEND}
+	test? ( sys-devel/sparse )
```

Comment:
Hmm, yes it seems to be some kind of test tool.
make check runs it like `sparse -O2 -pipe -march=znver4 -mshstk --param=l1-cache-line-size=64 --param=l1-cache-size=32 --param=l2-cache-size=1024 -std=gnu99 -Wall -Wextra -g -I. -DVERSION=\"5.1\" -DDO_DEBUG -I/usr/include/libnl3 *.[ch]`

I've put it into `BDEPEND`.
BTW, tests would fail for me because of:
` mach_clock_gettime.c:39:11: error: unable to open 'mach/clock.h'`
I guess there is some wrong/missing include somewhere, but since the ebuild has `RESTRICT="test"` anyway it's not really an issue?